lib/terminal.robot

```diff
@@ -153,8 +153,6 @@ Execute Command In Terminal
     Set DUT Response Timeout    ${timeout}
     IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
         ${output}=    Telnet.Execute Command    ${command}    strip_prompt=True
-    ELSE IF    '${DUT_CONNECTION_METHOD}' == 'SSH'
-        ${output}=    SSHLibrary.Execute Command    ${command}
```

Comment:
Wont removing this part breake tests for other machines?

Reply:
It might, I suppose.  It is required for testing the laptops though. I am not sure why it helps yet, I was instructed to use this change and it did help.
I thought this commit was already merged into develop. Either I imagined it or it was removed.

Reply:
Let's check if it does indeed break other machines, or maybe if this change is already in develop

Reply:
This change is already on the develop branch https://github.com/Dasharo/open-source-firmware-validation/blob/develop/lib/terminal.robot#L148-L163

Reply:
After rebasing the file does not change in this PR.